parmed/amber/_chamberparm.py

```diff
@@ -502,8 +502,10 @@ def _xfer_cmap_properties(self):
             flags_to_delete = [flag for flag in self.flag_list if flag.startswith('CHARMM_CMAP')]
             for flag in flags_to_delete:
                 self.delete_flag(flag)
-            del self.pointers['CMAP']
-            del self.pointers['CMAP_TYPES']
+            if 'CMAP' in self.pointers:
```

Comment:
I actually think `del self.pointers['CMAP']` is marginally clearer about the intent -- `pop` is something you use when you want to use an item after removing it from the collection.  `del` is what you use when you just want it removed.

Did you also know that `pop` is marginally slower than the other approach, even though it's only 1 line? :smiley:

```python
In [9]: S = 'Tribe'

In [10]: def popitem():
   ....:     d = {S: 1}
   ....:     d.pop(S, None)
   ....:     

In [11]: def delitem():
   ....:     d = {S: 1}
   ....:     if S in d:
   ....:         del d[S]
   ....:         

In [12]: %timeit popitem()
The slowest run took 14.37 times longer than the fastest. This could mean that an intermediate result is being cached 
1000000 loops, best of 3: 282 ns per loop

In [13]: %timeit delitem()
The slowest run took 16.98 times longer than the fastest. This could mean that an intermediate result is being cached 
1000000 loops, best of 3: 239 ns per loop

In [14]: def popitem():
   ....:     d = {}
   ....:     d.pop(S, None)
   ....:     

In [15]: def delitem():
   ....:     d = {}
   ....:     if S in d:
   ....:         del d[S]
   ....:         

In [16]: %timeit popitem()
The slowest run took 13.82 times longer than the fastest. This could mean that an intermediate result is being cached 
1000000 loops, best of 3: 224 ns per loop

In [17]: %timeit delitem()
The slowest run took 30.78 times longer than the fastest. This could mean that an intermediate result is being cached 
10000000 loops, best of 3: 163 ns per loop
```